File: pset6/mario.py
# CS50 Fall 2020
# Problem Set 6
# Author: kkphd

def prompt():
    while True:
        try:
            size = input("Height: ")
            converted = int(size)
            if converted > 0 and converted < 9:
                return converted
        except:
            print("Value must be a number.")

def blocks(height):
    for row in range(height):
        for space in range(height - row - 1):
            print(" ", end="")
        print("#" * (row + 1))

answer = prompt()
blocks(answer)

File: pset1/mario.c
// CS50 Fall 2020
// Problem Set 1
// Author: kkphd

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Initialize the height variable until it meets the conditions below.
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h > 8 || h < 1);

    // Create the loop for each row.
    for (int r = 0; r < h; r++)
    {
        // Create the loop for blank spaces and columns within each row.
        for (int s = 0; s < h - r - 1; s++)
        {
            printf(" ");
        }
        for (int c = 0; c <= r; c++)
        {
            printf("#");
        }

        printf("  ");

        for (int c = 0; c <= r; c++)
        {
            printf("#");
        }
        // Proceed to the next line.
        printf("\n");
    }
}

